deblur/test/test_workflow.py

```diff
@@ -522,6 +548,51 @@ def test_create_otu_table(self):
             'TACGTAGGTGGCAAGCGTTATCCGGAATTATTGGGCGTAAAGCGAGCGTAGGCGGTTTCTTA'
             'AGTCTGATGTGAAAGCCCACGGCTCAACCGTGGAGGGTCATTGGAAACTGGGGAACTTGAGT'
             'GCAGAAGAGGAGAGTGGAATTCCATGT', 'testmerge2'), 0)
+        # test the output fasta file
+        allseqs = []
+        with open(fasta_outfile, 'r') as input_f:
+            for label, seq in parse_fasta(input_f):
+                self.assertTrue(seq in tableids)
+                allseqs.append(seq)
+        self.assertEqual(len(allseqs), len(tableids))
+
+        # test minimal read filtering ( minreads>0 )
+        minreads = 7
+        outfile2 = join(self.working_dir, 'testmerge2.biom')
+        create_otu_table(outfile2, [(m1, 'testmerge'), (m2, 'testmerge2')],
+                         minreads=minreads)
+
+        table2 = load_table(outfile2)
+        table2ids = table2.ids(axis='observation')
+        tablesum = table.sum(axis='observation')
+        for idx, cid in enumerate(table.ids(axis='observation')):
+            if tablesum[idx] >= minreads:
+                self.assertTrue(cid in table2ids)
```

Comment:
I've never seen an assertContains.

But I know that there is an [assertIn](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIn)